recipes/configure-capture-agent-manager-nginx-proxy.rb

```diff
@@ -0,0 +1,38 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: configure-capture-agent-manager-nginx-proxy
+
+include_recipe "mh-opsworks-recipes::update-package-repo"
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+
+app_name = get_capture_agent_manager_app_name
+usr_name = get_capture_agent_manager_usr_name
+
+install_package("nginx")
+
+install_nginx_logrotate_customizations
+
+ssl_info = node.fetch(:ca_ssl, get_dummy_cert)
+if cert_defined(ssl_info)
+  create_ssl_cert(ssl_info)
+  certificate_exists = true
```

Comment:
It doesn't look like you're using `certificate_exists` anywhere, which makes me wonder if it would ever be the case that ssl would not be enabled? If not you might consider simplifying, e.g., the `install-kibana` recipe, where I just include the dummy cert in the default configuration.